spec/system/admin/reports/sales_tax/sales_tax_totals_by_order_spec.rb

```diff
@@ -466,13 +466,30 @@ def extract_xlsx_rows(file, range)
       end
 
       context "pdf files" do
+        let!(:report_file_pdf) do
+          PDF::Reader.new("spec/fixtures/reports/sales_tax_by_order/sales_tax_by_order.pdf")
+        end
+
         it 'downloads the file' do
           expect(downloaded_filenames.length).to eq(0) # downloads folder should be empty
           select "PDF", from: "report_format"
           click_on "Go"
           wait_for_download
           expect(downloaded_filenames.length).to eq(1) # downloads folder should contain 1 file
```

Comment:
I find this clearer:

```suggestion
          select "PDF", from: "report_format"

          expect {
            click_on "Go"
            wait_for_download
          }.to change {
            downloaded_filenames.length
          }.from(0).to(1)

```